pkg/specgen/generate/container.go

```diff
@@ -335,3 +336,145 @@ func FinishThrottleDevices(s *specgen.SpecGenerator) error {
 	}
 	return nil
 }
+
+func ConfigToSpec(rt *libpod.Runtime, specg *specgen.SpecGenerator, contaierID string) (*libpod.Container, error) {
```

Comment:
Please add a documentation comment as to what this does, I don't think it's immediately obvious